src/main/java/de/hysky/skyblocker/skyblock/calculators/SignCalculator.java

```diff
@@ -0,0 +1,59 @@
+package de.hysky.skyblocker.skyblock.calculators;
+
+import de.hysky.skyblocker.utils.Calculator;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.DrawContext;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+
+import java.text.DecimalFormat;
+
+public class SignCalculator {
+
+    private static final MinecraftClient CLIENT = MinecraftClient.getInstance();
+
+    private static final DecimalFormat FORMATTER = new DecimalFormat("#,###.##");
+
+    private static String lastInput;
+    private static String input;
+    private static Double output;
+
+    public static void renderSign(DrawContext context, String[] messages) {
+        input = messages[0];
```

Comment:
Passing a string array only to use the first one is kinda pointless in my opinion. Just pass a normal string